js-packages/@prestojs/ui-antd/src/widgets/SelectAsyncChoiceWidget.tsx

```diff
@@ -306,6 +315,27 @@ function SelectAsyncChoiceWidget<
         listOptions,
         retrieveOptions,
     });
+    const listRef = useRef(list);
+    listRef.current = list;
```

Comment:
why is this line necessary?

Reply:
yeah needs to be set every render. so the first time it will be initialised and then set to same value (i just did this so type on `useRef` doesn't have to include `null`)